plugins/vtd/bin/vtd-launch

```diff
@@ -247,6 +255,9 @@ def start(
     pane = window.attached_pane
     pane.clear()
 
+    if opt.log_file:
+        log_file = shlex.quote(str(opt.log_file))
+        pane.send_keys(f"script {log_file}")
```

Comment:
The `tmux` and `script` tools are very similar, there must be a better way than wrapping one in the other.

Reply:
That's right, it just took me a while to figure out. See the fixup.